Topics autopartitioning in CDC YDBDOCS-1104 (#18034)
ydb-platform/ydb: ydb/docs/en/core/concepts/cdc.md

## Topic Partitions {#topic-partitions}

> By default, the initial number of [topic partitions](topic.md#partitioning) is equal to the number of table partitions. The initial number of topic partitions can be redefined by specifying the [TOPIC_MIN_ACTIVE_PARTITIONS](../yql/reference/syntax/alter_table/changefeed.md) parameter when creating a changefeed. Changefeed with a dynamically changing number of partitions can be created by setting the [TOPIC_AUTO_PARTITIONING](../yql/reference/syntax/alter_table/changefeed.md) parameter when creating a changefeed.

Reviewer: ```suggestion
By default, the initial number of [topic partitions](topic.md#partitioning) is equal to the number of table partitions. You can redefine the initial number of topic partitions by specifying the [TOPIC_MIN_ACTIVE_PARTITIONS](../yql/reference/syntax/alter_table/changefeed.md) parameter when creating a changefeed. To create a changefeed with a dynamically changing number of partitions, set the [TOPIC_AUTO_PARTITIONING](../yql/reference/syntax/alter_table/changefeed.md) parameter when creating the changefeed.
```
